test/unit-test/CMakeLists.txt

```diff
@@ -109,12 +109,16 @@ create_test(ota_cbor_utest
 create_test(ota_os_posix_utest
     "ota_os_posix_utest.c"
     "${utest_link_list}"
-    "${utest_mem_ops_list}"
```

Comment:
Where was utest_mem_ops _list defined? Does that definition need to be removed if it's not being used anymore?

Reply:
It's not defined anywhere, so I'm removing it.